### PR TITLE
Add Loop market API for faster queries on Juno

### PR DESCRIPTION
**Pull request by noahsaso**

### CODE REVIEW

File: src/chains/cw721.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    if (err instanceof KnownError || err instanceof NotOwnerError) {
+      throw err;
+    }
+
+    console.error(err);
+    throw new KnownError(
+      500,
+      "Unexpected error retrieving NFT info from chain",
+      err
+    );
```
Comment (elsehow) [inline src/chains/cw721.ts:34]: Great error handling here.

File: src/chains/juno.ts [hunk 2]
```diff
@@ -1,10 +1,33 @@
...
+import { KnownError } from "../error";
 import { secp256k1PublicKeyToBech32Address } from "../utils";
-import * as Cw721 from "../cw721";
+import { ApolloClient, InMemoryCache, gql } from "@apollo/client";
+import { getOwnedNftImageUrl as makeCw721GetOwnedNftImageUrl } from "./cw721";
 
 const JUNO_RPC = "https://rpc.juno.strange.love:443";
+const LOOP_API_TEMPLATE = "https://nft-juno-backend.loop.markets";
+
+const GET_LOOP_NFTS_QUERY = gql`
```
Comment (elsehow) [inline src/chains/juno.ts:11]: What does a `gql`? Graphical query language (??). A comment could be helpful here to new contributors.

File: src/chains/juno.ts [hunk 3]
```diff
@@ -1,10 +1,33 @@
...
+  query GetLoopNfts($walletAddress: String!) {
+    nfts(filter: { owner: { equalTo: $walletAddress } }) {
+      nodes {
+        contractId
+        tokenID
+        image
+      }
+    }
+  }
+`;
```
Comment (elsehow) [inline src/chains/juno.ts:21]: Nice, this query is nicely self-documenting.

**Review by elsehow — APPROVED** [review]
Looks great!

File: src/chains/juno.ts (hunk 2 above)
Comment (noahsaso) [reply src/chains/juno.ts:11]: Graph Query Language :) Sure!

**Pull request merged**